INSTALLATION.md

```diff
@@ -338,9 +338,27 @@ GraphQL endpoint for handling `subscriptions` is this:
 
 ## For Mobile App Developers
 
-The Organization URL for Talawa mobile app developers to use is:
+The Organization URL for Talawa mobile app developers will depend upon the device on which Mobile app is installed.
 
-       http://localhost:4000/graphql/
+- If Talawa Mobile App is installed on Android Virtual Device (AVD): Use the following URL:
```

Comment:
Add a ### heading for this like:

### On a Virtual Device

Reply:
Done!